specs/brew/install_all_dependencies.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: Install all dependencies from Brewfile
+command: brew bundle install [--file={{file_location}}]
```

Comment:
shouldn't this be `brew bundle install  --file={{file_location}}`?

Reply:
No, as --file={{file_location}} is optional otherwise homebrew finds Brewfile from the current location.

Reply:
ah I seem workflows does not support the concept of optional values. This means that if the command is:
` brew bundle install [--file={{file_location}}]`, it will literally get inserted as that value into Warp

Reply:
Ok, moved this optional details to description.